submissions/TaniaBondarenko/friends-app/app.js

```diff
@@ -0,0 +1,131 @@
+const numOfFriends=24;
+const urlInit = `https://randomuser.me/api/?results=${numOfFriends}`;
+const friendsContainer = document.querySelector(".friends_container");
+let friendsData;
+let arrForFilter;
+let isFiltered = false;
+let justMale;
+let justFemale;
+
+function makeContainerEmpty() {
+        let innerText = " "; 
+        friendsContainer.innerHTML = innerText.replace(innerText, " ");    
+}
+
+function fetchFriend() {
+        fetch(urlInit)
+                .then(response => response.json())
+                .then((data) => {                         
+                        for (i = 0; i < numOfFriends; i++) {
+                                friendsData = data.results;
+                        }
+                })
+                .then(() => addFriends(friendsData))
+                .catch(err => {
+                        refreshPage();
+                });
+};
+
+function refreshPage() {
+        const wholePage = document.querySelector("body");
+        wholePage.setAttribute("class", "refresh");
+        wholePage.innerHTML = `<div>Ops...Something went wrong.</div>
+        <div>Refresh the page to start finding friends</div>
+        <button class="refresh_button">Click to refresh</button>` ;
+        document.querySelector(".refresh_button").addEventListener("click", ()=>
+                document.location.reload());
```

Comment:
Page reload is overkill. Please, handle this with your JS only

Reply:
Ok, I'll try